tests/php/Control/SessionHandler/FileSessionHandlerTest.php

```diff
@@ -308,26 +316,26 @@ public static function provideWrite(): array
         return [
             'overrides existing file' => [
                 'savePath' => static::$sessionSavePath,
-                'sessionFilePath' => Path::join(static::$sessionSavePath, FileSessionHandler::SESSION_FILE_PREFIX . 'test-existing-session'),
-                'sessionID' => 'test-existing-session',
+                'sessionFilePath' => Path::join(static::$sessionSavePath, FileSessionHandler::SESSION_FILE_PREFIX . FileSessionHandlerTest::ID_EXISTING),
+                'sessionID' => FileSessionHandlerTest::ID_EXISTING,
                 'createFileBeforeTest' => true,
             ],
             'overrides existing file with subdirs' => [
                 'savePath' => '2;' . static::$sessionSavePath,
-                'sessionFilePath' => Path::join(static::$sessionSavePath, 't/e', FileSessionHandler::SESSION_FILE_PREFIX . 'test-existing-session'),
-                'sessionID' => 'test-existing-session',
+                'sessionFilePath' => Path::join(static::$sessionSavePath, 'b/c', FileSessionHandler::SESSION_FILE_PREFIX . FileSessionHandlerTest::ID_EXISTING),
+                'sessionID' => FileSessionHandlerTest::ID_EXISTING,
                 'createFileBeforeTest' => true,
             ],
             'creates new file' => [
                 'savePath' => static::$sessionSavePath,
-                'sessionFilePath' => Path::join(static::$sessionSavePath, FileSessionHandler::SESSION_FILE_PREFIX . 'test-new-session'),
-                'sessionID' => 'test-new-session',
+                'sessionFilePath' => Path::join(static::$sessionSavePath, FileSessionHandler::SESSION_FILE_PREFIX . FileSessionHandlerTest::ID_EXISTING),
+                'sessionID' => FileSessionHandlerTest::ID_EXISTING,
```

Comment:
Looks like the wrong const here? (used to be new, const used is existing)

Reply:
Good catch, have fixed